components/DateForm.js

```diff
@@ -0,0 +1,103 @@
+import styled from "styled-components";
+import { StyledButton } from "./StyledButton.js";
+import { useState } from "react";
+
+const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 0.2rem;
+`;
+
+const Input = styled.input`
+  padding: 0.5rem;
+  border: 2px solid darkgreen;
+  border-radius: 0.5rem;
+`;
+
+const Textarea = styled.textarea`
+  border: 2px solid darkgreen;
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+`;
+
+const Label = styled.label`
+  font-weight: 400;
+`;
+
+export default function DateForm({ locationId, dates, setDates }) {
+  const formName = "test";
+
+  const [dateseries, setDateseries] = useState(false);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    const id = Math.floor(Math.random() * 1000);
+
+    setDates((draft) => {
+      draft.push({
+        id: id,
+        location: locationId,
+        data: data,
+      });
+    });
+  }
+
+  const handelToggleDateseries = (event) => {
+    setDateseries(event.target.value === "series");
+  };
+
+  return (
+    <>
```

Comment:
Hi Frank, 
in this case you dont need fragments, if a return includes two or more elements on highest level, than you can wrapp this in a fragment,
return (
<section />
<section />
)
this need a fragment or a wrapper.
Your <FormContainer > wrapps all elements and replaced a fragment